### DISCUSSION TITLE
Is there a plan to support floating in the future?

### DISCUSSION THREAD

**Discussion by Canoming** [typst/typst#849] [Q&A] [answered]
Seems the figures in Typst are placed as-where-it-is. Is there a plan to support the floats as in LaTeX? Typst may generate too many white spaces if figures are intermixed with long paragraphs now. Haven't figured out how to adjust the white spaces yet.

**Comment by LaurenzV** [comment, marked as answer]
It's been mentioned on Discord that floats will come. Not sure when though.

**Reply by Canoming** [reply]
Cool, thanks
